src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/Building/BuildingLambdaHandler.cs

```diff
@@ -3,12 +3,13 @@ namespace BuildingRegistry.Api.BackOffice.Handlers.Lambda.Handlers.Building
     using System.Configuration;
     using Be.Vlaanderen.Basisregisters.AggregateSource;
     using Be.Vlaanderen.Basisregisters.Api.ETag;
-    using BuildingRegistry.Api.BackOffice.Abstractions.Building.Responses;
+    using Be.Vlaanderen.Basisregisters.Sqs.Exceptions;
+    using Be.Vlaanderen.Basisregisters.Sqs.Lambda.Handlers;
+    using Be.Vlaanderen.Basisregisters.Sqs.Responses;
     using BuildingRegistry.Api.BackOffice.Abstractions.Building.Validators;
-    using Abstractions.Exceptions;
     using BuildingRegistry.Building;
     using BuildingRegistry.Building.Exceptions;
-    using BuildingRegistry.Infrastructure;
+    using Infrastructure;
     using MediatR;
     using Microsoft.Extensions.Configuration;
     using Requests.Building;
```

Comment:
Don't we want to inherit from SqsLambdaHandlerBase here? As was done in AddressRegistry?